css/site.styl

```diff
@@ -354,26 +354,35 @@ img
 // ******************************************************
 
 // HOME
-
-.home-item
-  width: 50%
-  text-align: center
-  .attachment-home-thumb
-    display: block
-  a
-    display: inline-block
-    cursor: pointer 
+#home-holder 
+  pointer-events: none
+  &:hover
+    .attachment-home-thumb
+      opacity: 0
+    .home-item-image-holder
+      border: $border
+  .home-item
+    pointer-events: auto
+    width: 50%
+    .attachment-home-thumb
+      display: block
+    a
+      display: inline-block
+      cursor: pointer 
     &:hover
+      .attachment-home-thumb
+        opacity: 1
       .home-item-image-caption
         opacity: 1
-      .attachment-home-thumb
-        opacity: 0
       .home-item-image-holder
-        border: $border
+        border: 3px solid transparent
+
```

Comment:
the problem with this hover hack (that changes all elements in the container in one way, and the hovered element in another) is that it triggers when on hovering `.home-item` which has some padding, so the trigger happens before the cursor is on the image.

i could do this with js for more precision. i like that this is pure css but it feels a little funny in this instance